From 5ff39230cf599ccff84756a4c8bcaa798e4c5d29 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 25 Oct 2022 20:33:39 +0100
Subject: [PATCH] configs: Enable the AHT10 sensor driver module

Add support for the Aosong AHT10 temperature and humidity sensor.

See: https://github.com/raspberrypi/linux/issues/5222

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 569cee8d4153..975e34d8cf5b 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -723,6 +723,7 @@ CONFIG_RPI_POE_POWER=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_MAX17040=m
 CONFIG_BATTERY_GAUGE_LTC2941=m
+CONFIG_SENSORS_AHT10=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_GPIO_FAN=m
 CONFIG_SENSORS_IIO_HWMON=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index c9e01d39ba18..fb707f2aa179 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -740,6 +740,7 @@ CONFIG_RPI_POE_POWER=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_MAX17040=m
 CONFIG_BATTERY_GAUGE_LTC2941=m
+CONFIG_SENSORS_AHT10=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_GPIO_FAN=m
 CONFIG_SENSORS_IIO_HWMON=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 80834a804779..c7b32a163807 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -717,6 +717,7 @@ CONFIG_RPI_POE_POWER=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_MAX17040=m
 CONFIG_BATTERY_GAUGE_LTC2941=m
+CONFIG_SENSORS_AHT10=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_GPIO_FAN=m
 CONFIG_SENSORS_IIO_HWMON=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 7724f0e9d6bd..c3209a4ed666 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -745,6 +745,7 @@ CONFIG_RPI_POE_POWER=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_MAX17040=m
 CONFIG_BATTERY_GAUGE_LTC2941=m
+CONFIG_SENSORS_AHT10=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_GPIO_FAN=m
 CONFIG_SENSORS_IIO_HWMON=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index e2a9970f0a56..eca03da7c431 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -716,6 +716,7 @@ CONFIG_RPI_POE_POWER=m
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_MAX17040=m
 CONFIG_BATTERY_GAUGE_LTC2941=m
+CONFIG_SENSORS_AHT10=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_GPIO_FAN=m
 CONFIG_SENSORS_IIO_HWMON=m
-- 
2.35.5

